anvil/src/winit.rs

```diff
@@ -218,6 +219,15 @@ pub fn run_winit(log: Logger) {
                 }
             }
 
+            // draw input method square if any
+            let input_method = state.seat.input_method();
+            let (x, y, _, height) = input_method.coordinates();
+            input_method.with_surface(|surface| {
+                if surface.alive() {
```

Comment:
Can we move the alive-check into `input_method.with_surface`? So that downstream does not have to deal with it.

Reply:
Fixed